tools/mummer4/promer_substitutions.py

```diff
@@ -31,12 +32,54 @@
 """
 
 FASTA_DIRECTORY = "fasta"
-FILTER_DIRECOTRY = "filter"
-SUBS_DIRECTORY = "substitutions"
+
+REF_START = 2
+REF_END = 3
 
 HEADER_ROW = "[P1]\t[SUB]\t[SUB]\t[P2]\t[BUFF]\t[DIST]\
     \t[R]\t[Q]\t[FRM]\t[FRM]\t[TAG]\t[TAG]\n"
 
+"""
+# =============================================================================
+REORIENT FILE
+# =============================================================================
+"""
+def reorient_file(fasta_location, start, end):
+
+    record = list(SeqIO.parse(fasta_location, "fasta"))[0]
+
+    # reversed
+    if start > end:
+        record.seq = record.seq[(end - 1):start].reverse_complement()
+
+    # same orientation
+    else:
+        record.seq = record.seq[(start - 1):end]
+
+    SeqIO.write(record, fasta_location, "fasta")
+
+
```

Comment:
If you only have two white spaces total between these messages, does the PEP8 error still occur?

Reply:
If I completely remove comments between the functions:

```
    # same orientation
    else:
        record.seq = record.seq[(start - 1):end]

    SeqIO.write(record, fasta_location, "fasta")


def promer(reference_location, query_location):
```

Then that error goes away. However, I would prefer to have multi-line comments either directly before or directly after the function definition.

Reply:
Would it suffice to have python docstrings instead of multi-line comments?
e.g.
```
def promer( ... ):
    """
       Function Explanation
    """

That way `help(promer)` will return your function explanation.

Reply:
It's possible to arrange the comment blocks as you suggest. However, I'm again reluctant to make the change because I don't prefer the docstring format for readability by a human.

Nonetheless, this isn't a hill I'm willing to die on and I'll change the code to meet the convention, despite my preferences.